main.js

```diff
@@ -67,26 +69,45 @@ function randomize() {
 }
 function displayMessage(message) {
   bellImage.classList.add('hidden');
+  deletedMessageOutput.classList.add('hidden');
   addMessageForm.classList.add('hidden');
-  messageSection.innerText = message;
+  messageOutput.innerText = message;
   clearMessageButton.classList.remove('hidden');
+  deleteMessageButton.classList.remove('hidden');
+
 }
 function clearMessage() {
-  if (bellImage.classList.contains('hidden')) {
+  addMessageForm.classList.add('hidden');
+  messageOutput.innerText = "";
+  clearMessageButton.classList.add('hidden');
+  deleteMessageButton.classList.add('hidden');
+  bellImage.classList.remove('hidden');
+}
+function deleteMessage() {
```

Comment:
This function is triggered when the Delete Message button is clicked. It first pops up a confirm window. If Ok is clicked, it then runs a for loop to find the message from the array, assign it to the deletedMessage variable, and then splice it from the array. Finally it displays a message saying that it was deleted.